tesseract_task_composer/planning/src/profiles/fix_state_collision_profile.cpp

```diff
@@ -45,6 +45,8 @@ FixStateCollisionProfile::FixStateCollisionProfile(Settings mode)
   collision_check_config.type = tesseract_collision::CollisionEvaluatorType::DISCRETE;
   trajopt_joint_constraint_config.coeff = Eigen::VectorXd::Constant(1, 1, 1);
   trajopt_joint_cost_config.coeff = Eigen::VectorXd::Constant(1, 1, 5);
+  collision_constraint_coeff = trajopt_common::CollisionCoeffData(1.0);
+  collision_cost_coeff = trajopt_common::CollisionCoeffData(20.0);
```

Comment:
Also need to add these to the constructor below for parsing from yaml file.

Reply:
Realized the yaml constructor was also missing the trajopt join configs. I also realized those did not have yaml encode/decode methods, so I have another PR to add that here: https://github.com/tesseract-robotics/trajopt/pull/469